packages/components/src/LightBox/LightBox.module.css

```diff
@@ -121,16 +118,81 @@
   max-width: 600px;
   max-height: 90px;
   margin: 0 auto var(--space-larger);
+  padding: 0 var(--space-base);
   overflow-y: auto;
-  text-align: center;
   flex-direction: column;
   flex-shrink: 0;
   gap: var(--space-small);
 }
 
+.title {
+  text-align: center;
+}
+
 .blurOverlay {
   position: fixed;
   inset: 0;
   backdrop-filter: blur(30px);
   background-color: rgba(3, 43, 58, 0.25);
 }
+
+.thumbnailBar {
+  display: flex;
+  position: relative;
+  z-index: var(--elevation-base);
+  width: 100%;
+  min-height: 90px;
+  margin-bottom: var(--space-small);
+  padding-bottom: var(--space-small);
+  overflow-x: scroll;
+  justify-content: safe center;
+  align-items: center;
+  gap: var(--space-small);
+  flex-shrink: 0;
+}
+
+.thumbnailImage {
+  width: 100%;
+  height: 100%;
+  border-radius: var(--radius-small);
+  opacity: 0.8;
```

Comment:
opacity overlay token please!

![image](https://github.com/user-attachments/assets/ed70c8aa-3fe3-454e-9965-77079d0d43cf)


unless there's a particular reason we don't want to have that matching